druid/src/core.rs

```diff
@@ -532,27 +532,26 @@ impl BaseState {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::widget::{Flex, IdentityWrapper, Scroll, Split, TextBox, WidgetExt};
-    use crate::{WidgetId, WindowId};
+    use crate::widget::{Flex, Scroll, Split, TextBox, WidgetExt};
+    use crate::WindowId;
+
+    const ID_1: WidgetId = WidgetId::reserved(0);
```

Comment:
This is a compromise; `next` cannot be used in a `const` context, so we have a separate constructor that is `const` compatible and which takes off the back of the allowed range, so that it will not collide with anything produced by `next`.

Reply:
I can see how `const` is more compatible with a global namespace.

Is it allowed to reuse the same widget id in multiple windows? (I'm thinking no, targeting is window specific). In that case, it seems to me it limits the use of these id's to a single window.

Reply:
you can reuse a widget id across windows, as long as it only exists in one place at a time; the same widget id cannot be used in multiple windows at once.